Makefile

```diff
@@ -84,6 +86,9 @@ define safe_rm_rf_children
 	bash -c 'test -z "$(1)" && false || rm -rf $(1)/*'
 endef
 
+## Default Task
+.DEFAULT_GOAL:=help
```

Comment:
maybe just stick the `help` goal here -- as I understand it, the first goal is the default in a makefile?